cloudbuild.yaml

```diff
@@ -14,9 +14,15 @@ steps:
 - name: gcr.io/cloud-builders/docker
   args: ['tag', 'bazel/java/jetty:jetty', 'gcr.io/$PROJECT_ID/java/jetty:latest']
 
+- name: gcr.io/cloud-builders/bazel
+  args: ['run', '//python2.7:python27']
+- name: gcr.io/cloud-builders/docker
+  args: ['tag', 'bazel/python2.7:python27', 'gcr.io/$PROJECT_ID/python2.7:latest']
+
 # The resulting images to push to GCR.
 images: [
   'gcr.io/$PROJECT_ID/base:latest',
   'gcr.io/$PROJECT_ID/java:latest',
   'gcr.io/$PROJECT_ID/java/jetty:latest',
+  'gcr.io/$PROJECT_ID/python2.7:latest'
```

Comment:
nit: we put 2.7 in the name for python, but not for java.  I understand Python's weirder, but we should consider how we might resolve this.